com.seventhstring.Transcribe.yml

```diff
@@ -0,0 +1,61 @@
+app-id: com.seventhstring.Transcribe
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/transcribe/transcribe
```

Comment:
this is never an absolute path. Just create a symlink in `/app/bin`

Reply:
done